src/neuroconv/datainterfaces/behavior/deeplabcut/_dlc_utils.py

```diff
@@ -58,14 +58,11 @@ def _get_movie_timestamps(movie_file, VARIABILITYBOUND=1000, infer_timestamps=Tr
     n_frames = int(reader.get(cv2.CAP_PROP_FRAME_COUNT))
     fps = reader.get(cv2.CAP_PROP_FPS)
 
+    warnings.warn("Inferring timestamps from video. This might take a while (to speed up, set timestamps)")
     for _ in range(n_frames):
         _ = reader.read()
         timestamps.append(reader.get(cv2.CAP_PROP_POS_MSEC))
 
-    for _ in range(len(reader)):
```

Comment:
Can you explain why you removed this? You mention it solves a bug, which bug and how.

Reply:
reader object does not have a `__len__` (maybe it had in previous `cv2` versions?). So I replaced it with `n_frames`. I did not understand how both loops ended up there, I suspect a merging issue.